server/utils/utils.ts

```diff
@@ -21,3 +23,30 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
+export const switchDateFormat = (displayDate: string, fromFormat = 'dd/MM/yyyy') => {
+  if (displayDate) {
+    return DateTime.fromFormat(displayDate, fromFormat, { locale: 'en-GB' }).toISODate()
+  }
+  return displayDate
+}
+
+export const getCurrentPeriod = (dt: DateTime): string => {
+  const afternoonSplit = 12
+  const eveningSplit = 17
+  if (dt.hour < afternoonSplit) return 'AM'
+  if (dt.hour < eveningSplit) return 'PM'
+  return 'ED'
+}
```

Comment:
I know moment.js would do this for free, does Luxor have an equivalent?

Reply:
or `date-fns`?

Reply:
Logic is customised for this app - 'ED'

Reply:
What is ED? Is that a concept introduced by us then?

Reply:
DPS seems to use it - This first PR was just a lift of the DPS form for choosing activity location - I guess activities after 5pm are classed as ED